src/@chakra-ui/gatsby-plugin/components/Text.ts

```diff
@@ -0,0 +1,48 @@
+import { defineStyle, defineStyleConfig } from "@chakra-ui/react"
+
+const sizes = {
```

Comment:
@pettinarip to clarify, this is just regarding the `Text` component? It looks like the sizes for the `Heading` component have not changed (which is to be expected since they are currently inline with the Chakra defaults)

Reply:
Also need clarification regarding the body copy with this update. Marked a question [in the figma file](https://www.figma.com/file/NrNxGjBL0Yl1PrNrOT8G2B?node-id=384:15485&mode=dev#520717779) as I found a discrepancy.

Reply:
Correct @TylerAPfledderer. The `Heading` has not suffered any update.